security-manager/src/main/java/org/wildfly/extension/security/manager/SecurityManagerSubsystemAdd.java

```diff
@@ -109,10 +108,11 @@ protected void launchServices(final OperationContext context, final ModelNode no
                 this.peek(node, DEPLOYMENT_PERMISSIONS, DEFAULT_VALUE, MINIMUM_PERMISSIONS));
 
         // get the maximum set of deployment permissions.
-        final List<PermissionFactory> maximumSet = this.retrievePermissionSet(context,
-                this.peek(node, DEPLOYMENT_PERMISSIONS, DEFAULT_VALUE, MAXIMUM_PERMISSIONS));
+        final ModelNode maximumSetNode = this.peek(node, DEPLOYMENT_PERMISSIONS, DEFAULT_VALUE, MAXIMUM_PERMISSIONS);
```

Comment:
This peek here, should be using AttributeDefinitions as parameters not just strings of the names of attributes.
With AD you could properly resolve the model